### PR TITLE
Support sending and receiving stickers for WAC and D3C

### PR DESCRIPTION
**Pull request by norkans7**
This adds support to send and receive stickers as documented on https://developers.facebook.com/docs/whatsapp/cloud-api/reference/media#supported-media-types

The file need to be for format `image/webp` and size less than 100KB or 500KB for animated one with dimensions not exceeding 512x512

### CODE REVIEW

**Comment by rowanseymour** [comment]
@norkans7 since this isn't a bug gonna hold off merging until after release

File: handlers/dialog360/dialog360.go [hunk 1]
```diff
@@ -621,7 +632,9 @@ func (h *handler) Send(ctx context.Context, msg courier.Msg, clog *courier.Chann
 
 		} else if i < len(msg.Attachments()) && (len(qrs) == 0 || len(qrs) > 3) {
 			attType, attURL := handlers.SplitAttachment(msg.Attachments()[i])
-			attType = strings.Split(attType, "/")[0]
+			splitedAttType := strings.Split(attType, "/")
```
Comment (rowanseymour) [inline handlers/dialog360/dialog360.go:635]: @norkans7 how about we rework this to call resolve media and then we can also check that the image is 512 x 512 ?

Comment (norkans7) [reply handlers/dialog360/dialog360.go:635]: Okay, I'll update that

**Comment by norkans7** [comment]
contains #624

**Comment by norkans7** [comment]
This branch is out of date I will rework on that again

**Pull request closed without merging**